src/helm/benchmark/static/benchmarking.js

```diff
@@ -1045,17 +1045,63 @@ $(function () {
     return $('<td>').append($linkedValue);
   }
 
+  function renderTableHeader(table) {
+    const $tableHeader = $('<thead>');
+    const $row = $('<tr>').append(table.header.map((cell, index) => {
+      $cell = renderCell(cell);
+      // Use the arrow characters to determine the sort order
+      // Ideally the schema would be used instead, but the schema is not available here
+      const sortOrder = cell.value.includes("\u2191") ? "desc" :
+        (cell.value.includes("\u2193") ? "asc" : "");
+      if (sortOrder) {
+        const $sortLink = $("<a>", {"href": "#"}).append("sort").click(() => {
+          $table = $tableHeader.parent('table');
+          $table.find('tbody').remove();
+          $table.append(renderTableBody(table, index, sortOrder));
+          return false;
+        });
+        $cell
+          .append(" [&nbsp;")
+          .append($sortLink)
+          .append("&nbsp;]");
```

Comment:
Is there a way to make this look less clunky?

Reply:
Do you mean in UI? Options are:

- Use a [sort icon](https://thenounproject.com/browse/icons/term/sort/?iconspage=1) (e.g. an arrow)
  - Pros: Visually pleasing.
  - Cons: Small click target. The cell text already has an arrow so we would need to remove it to have not have two arrows (or we could turn the existing arrow in the cell text into a click target, but the arrow is sometimes in the middle of the text).
- Make the whole column text a click target
  - Pros: Saves space; large click target.
  - Cons: Clickability is not as clearly communicated.

Reply:
We need a way to communicate when a column has been clicked and is sorted by.  Shading?  Once this happens, I think we could go with #2.